src/kirby/components/modal/modal.component.ts

```diff
@@ -9,14 +9,34 @@ import { IModalController } from './services/modal.controller.interface';
   templateUrl: './modal.component.html',
   styleUrls: ['./modal.component.scss'],
 })
-export class ModalComponent {
+export class ModalComponent implements AfterViewInit {
+  @ViewChild('modalWrapper') modalWrapper: ElementRef;
+  scrollY: number = Math.abs(window.scrollY);
```

Comment:
Can't this be really expensive? I wonder how many of these events are emitted for a single scroll down the page.

Reply:
I wouldn't say so. This is the position of the scroll at the time when we clicked on the modal-opening button. We compute it only once and then use it inside the class, more precisely the value is used every time the focus has changed (which happens when we press tab on the keyboard, or click on something with the mouse). With this said, I am confused about which events you refer to, maybe I am not seeing something.

Reply:
Wow, sorry. I'm tired today. I thought you subscribed to scroll events.